lib/rubocop/config_loader.rb

```diff
@@ -58,7 +58,11 @@ def merge(base_hash, derived_hash)
 
       def base_configs(path, inherit_from)
         configs = Array(inherit_from).map do |f|
-          f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          if f.include?('~')
+            f = File.expand_path(f)
+          else
+            f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          end
```

Comment:
Hmm, I just tried `File.expand_path(file, path)` and then still does not work. The error message still says no file exists.